torch/csrc/distributed/c10d/FlightRecorder.hpp

```diff
@@ -12,6 +12,7 @@
 #include <ATen/ATen.h>
 #include <ATen/cuda/CUDAEvent.h>
 #include <c10/util/Exception.h>
+#include <nlohmann/json.hpp>
```

Comment:
I guess this header is not necessary?

Reply:
Oh the next PR needs it.. let me update..